test/scss/_main.test.scss

```diff
@@ -7,9 +7,10 @@
 						@include oAutocomplete();
 					}
 				}
-				@include expect($selector: false) {
-					.test-mixin .o-autocomplete {
-						display: block;
+				@include contains($selector: false) {
```

Comment:
Question: Does this component need sass tests?

Reply:
I would say not, since there is only one primary mixin with no options -- Percy can catch visual changes